### PR TITLE
Some code refactoring following pull request #124

### PR DESCRIPTION
**Pull request by liyinan926**
Signed-off-by: Yinan Li <liyinan926@gmail.com>

### CODE REVIEW

**Comment by liyinan926** [comment]
@ibuenros, can you review this?

File: gobblin-runtime/src/main/java/gobblin/runtime/TaskContext.java [hunk 1]
```diff
@@ -251,9 +259,10 @@ public TaskPublisher getTaskPublisher(TaskState taskState, TaskLevelPolicyCheckR
    */
   public DataWriterBuilder getDataWriterBuilder(int branches, int index) {
     try {
-      return (DataWriterBuilder) Class.forName(this.workUnit
-          .getProp(ForkOperatorUtils.getPropertyNameForBranch(ConfigurationKeys.WRITER_BUILDER_CLASS, branches, index),
-              ConfigurationKeys.DEFAULT_WRITER_BUILDER_CLASS)).newInstance();
+      String dataWriterBuilderClassName = this.workUnit.getProp(
+          ForkOperatorUtils.getPropertyNameForBranch(ConfigurationKeys.WRITER_BUILDER_CLASS, branches, index),
+          ConfigurationKeys.DEFAULT_WRITER_BUILDER_CLASS);
+      return (DataWriterBuilder) Class.forName(dataWriterBuilderClassName).newInstance();
```
Comment (sahilTakiar) [inline gobblin-runtime/src/main/java/gobblin/runtime/TaskContext.java]: Minor comment, but if you want to split the `return` line into multiple lines, then you can move the `String dataWriterBuilderClassName = ...` declaration outside of the try, catch block.

**Comment by sahilTakiar** [comment]
Minor comment, otherwise LGTM

**Comment by liyinan926** [comment]
Updated d029165.

File: gobblin-runtime/src/main/java/gobblin/runtime/TaskContext.java (hunk 1 above)
Comment (liyinan926) [inline gobblin-runtime/src/main/java/gobblin/runtime/TaskContext.java]: Fixed.

**Pull request merged**